school/diary/views.py

```diff
@@ -47,42 +47,23 @@ def index(request):
     return HttpResponse(index_template)
 
 
-# def login(request):
-#     if request.POST:
-#         name = request.POST.get("name")
-#         position = request.POST.get("position")
-#         club = request.POST.get("contact")
-#         transfer_fee = request.POST.get("transfer_fee")
-#         response = f"Player Name: {name}, Position: {position}, Club: {club}, " \
-#                    f"Transfer fee: {transfer_fee} millions euros"
-#     else:
-#         response = "Please create a player"
-#     return HttpResponse(response)
-
 def login(request):
```

Comment:
Great! What about endpoints for changing the params we had previously in Flask app?

Reply:
You mean endpoints that gives you an opportunity to change some existing parameters in DB?

Reply:
correct

Reply:
if i knew how(

Reply:
parameters can be changed via form?